src/main/java/somethingrandom/dataaccess/google/tasks/GoogleTasksDataAccessObject.java

```diff
@@ -32,4 +34,13 @@ public void save(Item item) throws DataAccessException {
             throw new DataAccessException(e);
         }
     }
+
+    @Override
+    public Collection<Item> getAll() {
```

Comment:
Can you rename this to getAllItems()? That's the name in the Search Data Access Interface